tutorial/server/axum/src/main.rs

```diff
@@ -0,0 +1,150 @@
+use axum::{
+    extract::Extension,
+    response::{Html, IntoResponse},
+    routing::{get, post},
+    Router,
+};
+use axum_sessions::{async_session::MemoryStore, SameSite, SessionLayer};
+use std::net::SocketAddr;
+mod error;
+mod file;
+/*
+ * Webauthn RS server side tutorial.
+ */
+
+// 1. Import the prelude - this contains everything needed for the server to function.
+// use webauthn_rs::prelude::*;
+
+// These are other imports needed to make the site generally work.
+
+use crate::auth::{finish_authentication, finish_register, start_authentication, start_register};
+use crate::file::file_handler;
+use crate::startup::AppState;
+
+use rand::prelude::*;
+
+#[macro_use]
+extern crate tracing;
+
+mod auth;
+mod startup;
+
+// 7. That's it! The user has now authenticated!
```

Comment:
Seems a bit out of place?

Reply:
That's what I thought, but the tide version has it in the same spot above index_view

Reply:
Ahhhh I see. :)